### DISCUSSION TITLE
Email not sent

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6869] [Help]
Hi @VictorTomaili 

Is it possible that email service works only with StartSharp.

I'm not able to send email on serenity, for example in _Forgot Password_ process.

Mails are generated and saved in configured folder (as defined in appsettings.json)

I configured smtpsetting in all appsettings file, to be sure that the problem was not the environment (see attached images)

During debug, the smtpsetting are all null; I do not understand why! (se attached video)

Can you help me?

Thank you,
Elena.

![Schermata 2023-08-07 alle 10 23 21](https://github.com/serenity-is/Serenity/assets/14870650/51327179-0685-4681-a0c6-22670ee5ef08)

![Schermata 2023-08-07 alle 10 22 54](https://github.com/serenity-is/Serenity/assets/14870650/36d56a9f-28c8-43df-a04b-a5aeca796b2f)

![Schermata 2023-08-07 alle 10 22 26](https://github.com/serenity-is/Serenity/assets/14870650/ac54ba18-07fc-4103-842e-f22ec71df0ea)

https://github.com/serenity-is/Serenity/assets/14870650/cbbe6697-5669-4aa4-987a-a75f52d4f925

**Comment by elenaaralla** [comment]
Hi all,

adding this code in startup.cs file:

`services.Configure<SmtpSettings>(Configuration.GetSection(SmtpSettings.SectionKey));`

now I can send email.

Now, the question is: where this settings are documented?  :)

I hope this helps some one else.

Thank you,
Elena.

**Comment by nahidulislam-cse15** [comment]
what configuration need for my local smstp server to send link of resert password ,i can send sig up email but failed to reset password

**Comment by nahidulislam-cse15** [comment]
when i changed in appsetting it says  TLS 4.7 error  on local
